erpnext_com/www/docs/user/manual/en/using-erpnext/articles/keyboard-shortcuts.md

```diff
@@ -3,35 +3,36 @@
 
 ERPNext Version 12 is almost fully accessible using keyboard shortcuts.
 
-####Keyboard Shortcuts Dialog
+##Keyboard Shortcuts Dialog
 
-The keyboard shortcuts for any page can be viewed by opening the keyboard shortcuts dialog from the Help dropdown on the navbar.
+The keyboard shortcuts for any page can be viewed by opening the keyboard shortcuts dialog from the Help dropdown on the navigation bar.
 
-<img alt="Keyboard Shortcuts link" class="screenshot" src="{{docs_base_url}}/assets/img/articles/keyboard-shortcuts-link.png">
+<img alt="Keyboard Shortcuts link" class="screenshot" src="{{docs_base_url}}/assets/img/collaboration-tools/keyboard-shortcuts-link.png">
```

Comment:
small detail but, the red box should border the gray area edge to edge, don't leave space when highlighting